javatests/arcs/core/analysis/TypeConstraintsTest.kt

```diff
@@ -0,0 +1,158 @@
+package arcs.core.analysis
+
+import arcs.core.data.HandleConnectionSpec
+import arcs.core.data.Recipe
+import arcs.core.data.Recipe.Handle
+import arcs.core.data.Recipe.Particle
+import arcs.core.data.Recipe.Particle.HandleConnection
+import arcs.core.data.ParticleSpec
+import arcs.core.data.TypeVariable
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.fail
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+typealias Direction = HandleConnectionSpec.Direction
+
+/** Tests for TypeConstraints. */
+@RunWith(JUnit4::class)
+class TypeConstraintsTest {
+    // ---- The test environment ----
+    //
+    //  particle ConnectionsTest
+    //    text_cnxn: reads
+    //    word_cnxn: reads
+    //    num_cnxn: reads
+    //
+    //  particle TypeVariablesTest
+    //    num_cnxn: reads TypeVariable("num_type")
+    //    int_cnxn: reads TypeVariable("num_type")
+    //
+    var ramdiskStorageKey = "ramdisk://things"
+    val textHandle = Handle(
+        "text", Handle.Fate.CREATE, ramdiskStorageKey, TypeVariable("text"), emptyList()
```

Comment:
One arg per line please

(I believe the style we follow is, unless everything fits on the same line, use one arg per line)

Reply:
Thanks for the info. I wasn't sure if it was OK to have all parameters in one line. Changed it throughout the file.